front/src/components/app/App.js

```diff
@@ -1,10 +1,18 @@
 import React, { Component } from 'react';
 import ExampleForRedux from '../exampleForRedux/ExampleForRedux';
```

Comment:
Do we need this import anymore, please, remove this component if it is not used anywhere

Reply:
done